frontend/scripts/react-components/home/entrypoints.component.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import Siema from 'react-siema';
+import Link from 'redux-first-router-link';
+import debounce from 'lodash/debounce';
+
+class Entrypoints extends React.PureComponent {
+  static getIsMobile() {
+    return window.innerWidth <= 700;
```

Comment:
not sure if that's a correct way, but I'm always using breakpoint foundation small width.

Reply:
I always try to use that one but in this case the content break before, so added a cuatom one